.rubocop_todo.yml

```diff
@@ -1,15 +0,0 @@
-# This configuration was generated by
-# `rubocop --auto-gen-config`
-# on 2020-06-30 20:30:26 -0700 using RuboCop version 0.75.1.
-# The point is for the user to remove these configuration records
-# one by one as the offenses are removed from the code base.
-# Note that changes in the inspected code, or installation of new
-# versions of RuboCop, may require this file to be generated again.
-
-# Offense count: 1
----
-ChefCorrectness/NodeNormal:
```

Comment:
This file needs to be added back so cookstyle doesn't warn about `node.normal` use in the recipes/setup_shcluster.rb. In a normal use case, `node.normal` isn't recommended; however, there is a need to preserve an attribute value between chef runs. This is to prevent a bootstrap captain in a search head cluster from being bootstrapped once a captain has already been bootstrapped.

Reply:
I setup an inline ignore to fix the cookstyle error.

Reply:
reverted my changes and moved my changes to a new fork TheREK3R#1